lesson_09-HW_19/index.js

```diff
@@ -0,0 +1,74 @@
+class Person {
+   name;
+   age;
+
+   constructor(name, age) {
+      this.name = name;
+      this.age = age;
+   }
+}
+
+class Flat {
+   residents = [];
+
+   addResidents(person) {
```

Comment:
```suggestion
   addResident(person) {
```
Ви ж їх по одному додаєте